src/ctest.ts

```diff
@@ -426,10 +294,24 @@ export class CTestDriver implements vscode.Disposable {
                 this.ctestErrored(child, run, message);
             }
         } else {
+            if (test.uri && test.range) {
+                message.location = new vscode.Location(test.uri, test.range);
+            } else {
+                log.error(message);
```

Comment:
For my own understanding, are we only logging errors that don't have a URI and range because errors that have a URI and range will be directly displayed in the VS Code UI?